build.gradle.kts

```diff
@@ -1,22 +1,39 @@
 import com.fortysevendeg.hood.CompareBenchmark
+import com.jfrog.bintray.gradle.BintrayExtension
 
 plugins {
   kotlin("jvm") version "1.3.20"
   id("com.fortysevendeg.hood")
+  id("com.jfrog.bintray") version "1.8.4"
 }
 
 repositories {
+  jcenter()
```

Comment:
Be careful with using `jcenter` as first one, it should be `mavenCentral`: https://twitter.com/jakewharton/status/1073102730443526144?lang=en